timesketch/api/v1/resources/upload.py

```diff
@@ -475,6 +476,8 @@ def post(self):
 
         # headers mapping: map between mandatory headers and new ones
         headers_mapping = json.loads(form.get("headersMapping", "{}")) or None
+        logger.info(headers_mapping)
```

Comment:
do we need this moving forward or is this more a debug message, if we leave it, I opt to add more text around what it is.